src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/WhereParser.java

```diff
@@ -506,7 +506,7 @@ private void explanCond(String opear, SQLExpr expr, Where where) throws SqlParse
 
             where.addWhere(condition);
         } else {
-            throw new SqlParseException("err find condition " + expr.getClass());
+            throw new SqlParseException("error parsing condition for " + expr.getClass());
```

Comment:
Thanks for fixing the typo and error message! Could you return `expr.getClass().getSimpleName()`? I recall I saw long Druid class name returned.